manualTests/src/main/x/TestMisc.x

```diff
@@ -3,12 +3,12 @@ module TestMisc {
 
     void run() {
         console.print("hello world!");
-
+        
         testBools();
-        testChars();
+        //testChars();
+        testAsciiDigit();
```

Comment:
This is the standard master testmisc, and merging it will have the effect of whatever is in testChars() and the other connected out methods will not be run from master anymore. Is that what we want?